### ISSUE TITLE
google_cloudfunctions_function doesn't support changing docker registry to use when building the function

### ISSUE THREAD

**Issue by eslamkarim** [hashicorp/terraform-provider-google#11341]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment. If the issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If the issue is assigned to a user, that user is claiming responsibility for the issue. If the issue is assigned to "hashibot", a community member has claimed the issue already.

### New or Affected Resource(s)

* google_cloudfunctions_function

### Potential Terraform Configuration

```tf
resource "google_cloudfunctions_function" "function" {
  name        = "function-test"
  description = "My function"
  runtime     = "nodejs14"
  available_memory_mb   = 128

  dockerRepository = "cutome-repo"
...
}
```

### References

* b/274818266

**Comment by fleetwoodstack** [comment]
Is this still relevant? Isn't the field called `docker_repository` and not `dockerRepository `

**Comment by eslamkarim** [comment]
yes this is not relevant anymore since then this feature has been added to the provider.
thank you.
